### PR TITLE
style(TeaserMyMagazie): design tweaks teasermymagazine

### PR DESCRIPTION
**Pull request by ovbm**
- Schriftarten Titel angepasst
- Grauer Rahmen
- Layout angepasst, einheitlich mit dialog

Screenshot 2020-10-27 at 15 27 35

### CODE REVIEW

File: src/components/TeaserMyMagazine/TeaserMyMagazine.js [hunk 1]
```diff
@@ -120,6 +124,11 @@ const TeaserMyMagazine = ({
                   </div>
                 )
               })}
+              <Link href={bookmarksUrl} passHref>
+                <TeaserSectionTitle small href={bookmarksUrl}>
+                  {'Alle Beiträge zum Weiterlesen'}
```
Comment (tpreusse) [inline src/components/TeaserMyMagazine/TeaserMyMagazine.js:129]: Let's use `bookmarkLabel`

File: src/components/TeaserMyMagazine/TeaserMyMagazine.js [hunk 2]
```diff
@@ -51,20 +52,14 @@ const TeaserMyMagazine = ({
       }}
     >
       <section {...css(styles.section)}>
+        <TeaserSectionTitle>Meine Republik</TeaserSectionTitle>
```
Comment (tpreusse) [inline src/components/TeaserMyMagazine/TeaserMyMagazine.js:55]: Let's make this Publikator configurable: use a `title` prop. And forward it in `templates/Front/liveTeasers`—just like `bookmarkLabel`.

While at it please also add the following form config to the `editorOptions` of `type: 'LIVETEASERMYMAGAZINE'`:
```
        form: [
          {
            key: 'title'
          },
          {
            key: 'bookmarkLabel'
          },
          {
            key: 'bookmarksUrl'
          },
          {
            key: 'notificationsLabel'
          },
          {
            key: 'notificationsUrl'
          }
        ]
```
This ensures that those fields are always editable in Publikator. And let's also fix the inconsistency with `bookmarkLabel` and `bookmarksUrl`  🙈

Comment (ovbm) [reply src/components/TeaserMyMagazine/TeaserMyMagazine.js]: done!

**Review by tpreusse — APPROVED** [review]
Code looks good. Submitted it to CR for feedback. Let's wait for that before merging

**Pull request merged**